astroquery/sdss/core.py

```diff
@@ -1072,5 +1075,55 @@ def _get_crossid_url(self, data_release):
         self._last_url = url
         return url
 
+    def view_in_sdss_navigate(self, sc):
+        """
+        To view "Navigate" given the SkyCoord sc of the point of interest.
+
+        Parameters
+        ----------
+        sc: SkyCoord of a point
+
+        Returns
+        -------
+        Response: Open a browser tab in the SDSS "Navigate" interface
+
+        """
+        if isinstance(sc, SkyCoord):
+            if sc.shape == ():
+                payload = {"ra": str(Angle(sc.ra).degree), "dec": str(Angle(sc.dec).degree), "opt": "GO"}
+                r = requests.get("http://skyserver.sdss.org/dr15/en/tools/chart/navi.aspx", params=payload)
+                webbrowser.open_new_tab(r.url)
+            else:
+                raise TypeError("The given SkyCoord object is not of the right shape.")
```

Comment:
This message could be more helpful.  e.g., "Only scalar SkyCoordinates are allowed." or "Arrays of coordinates are not supported by <this function>"

Reply:
Yup, changed error message to "Only scalar SkyCoordinates are allowed."